app/models/settlement.rb

```diff
@@ -7,7 +7,7 @@ class Settlement < ActiveRecord::Base
     'instnm' => { column: :institution, converter: InstitutionConverter },
     'school_system_code' => { column: :school_system_code, converter: NumberConverter },
     'school_system_name' => { column: :school_system_name, converter: BaseConverter },
-    'settlement_description' => { column: :settlement_description, converter: DisplayConverter },
+    'settlement_description' => { column: :settlement_description, converter: BaseConverter },
```

Comment:
Patrick's issue was with the case of letters in acroynyms being an issue. The BaseConverter leaves the case intact, but does strip quote characters, null, none and privacysuppressed, as well leading and trailing blanks. We are now displaying caution flag reasons exactly as input.